platforms/ide/tooling-api/src/test/groovy/org/gradle/integtests/tooling/fixture/ToolingApiDistributionResolverTest.groovy

```diff
@@ -40,15 +41,17 @@ class ToolingApiDistributionResolverTest extends Specification {
 
     ToolingApiDistributionResolver underTest = new ToolingApiDistributionResolver()
 
+    @RestoreSystemProperties
     def "uses distribution from classpath when resolving current version"() {
         given:
+        System.setProperty("toolingApi.shadedJar", "/path/to/shaded.jar")
```

Comment:
❓could we remove the property or set it to a relative path like `does-not-exist/something.jar`?